main.go

```diff
@@ -88,6 +88,23 @@ func main() {
 		},
 	)
 
+	if vargs.UpdateEnvironment == true {
```

Comment:
why do I need error check before that if block? only if there is UpdateEnvironment arg then I check EnvironmentName is there I don't need to know that beforehand.

Also why not utilize the same err why use a new one it will create a duplicate